app/models/spree/gateway/amazon.rb

```diff
@@ -135,5 +146,11 @@ def void(response_code, gateway_options)
     def load_amazon_mws(reference)
       @mws ||= AmazonMws.new(reference, gateway: self)
     end
+
+    # A random string of lowercase alphanumeric characters (i.e. "base 36")
+    def random_suffix
```

Comment:
Should we just do: `SecureRandom.hex(5)`? Seems like the end result would be the same